addons/mail/static/src/models/thread/thread.js

```diff
@@ -1546,11 +1546,14 @@ function factory(dependencies) {
                 // avoid crash during destroy
                 return;
             }
+            if (this.env.messaging.device.isMobile) {
+                return;
+            }
             if (this.serverFoldState === 'closed') {
                 this.env.messaging.chatWindowManager.closeThread(this, {
                     notifyServer: false,
                 });
-            } else {
+            } else if (!this.env.messaging.device.isMobile) {
```

Comment:
You added the same conditions 6 lines above, no need to duplicate it here.

Reply:
Done